static/styles/left_sidebar.css

```diff
@@ -20,7 +20,7 @@ $topic_resolve_width: 13px;
         &::after {
             content: "#";
             line-height: 0;
-            font-size: 18px;
+            font-size: 17px;
```

Comment:
This change should only be applied to the `hastag` in compose box, not across the app.